### PR TITLE
Docker Compose setup for new Institution Search

### PR DESCRIPTION
**Pull request by hkeeler**
This is a Docker Compose update that goes along with https://github.com/cfpb/hmda-platform-auth/pull/75.  **These 2 PR should be merged at the same time**.

This PR is also dependent on #798 being merged first.

### CODE REVIEW

File: Documents/public-api.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+              "value": "1234",
+              "name": "occ-charter-id"
+            },
+            {
+              "value": "1234",
+              "name": "ncua-charter-id"
+            }
+          ]
+        }
+     ]
```
Comment (awolfe76) [inline Documents/public-api.md]: Missing `}`

Comment (hkeeler) [inline Documents/public-api.md]: That wasn't actually part of this PR.  It came from #798...but I fixed it here (https://github.com/cfpb/hmda-platform/pull/808/commits/31426276a9c2239c69061fa37e9675393591786c) anyway, just for you @awolfe76! :)

**Comment by hkeeler** [comment]
I've merged in the changes from PR #798, done the necessary conflict fixups, and add a small doc fix.  This should be good to go as soon as https://github.com/cfpb/hmda-platform-auth/pull/75 is merged.

**Comment by nickgrippin** [comment]
@hkeeler: I can merge this in as soon as the other one is merged.

**Comment by jmarin** [comment]
LGTM

**Pull request merged**